modules/generate/src/main/scala/render.scala

```diff
@@ -636,7 +636,12 @@ class Render(manager: Manager, packageName: String = "langoustine.lsp"):
         case other =>
           val newType = other.traverse {
             case stl: Type.StructureLiteralType =>
-              val newTypeName = "S" + inlineAnonymousStructures
+              val newTypeName = summon[Config].anonNameOverrides
+                .get(structure.name.value)
+                .fold("S" + inlineAnonymousStructures)(
```

Comment:
technically these `S` names don't happen anymore, but this is here to account for new types being added in future LSP versions and the maintainer not wanting to give them names immediately.

Reply:
Actually I think it's worth being strict here – better to fix the names once during update, than to expose `S*` and then have to break source compat renaming them.

Reply:
not completely unreasonable